webapp/src/components/chat/ChatRoom.tsx

```diff
@@ -105,6 +106,22 @@ export const ChatRoom: React.FC = () => {
         setShouldAutoScroll(true);
     };
 
+    if (conversations[selectedId].hidden) {
+        return (
+            <div className={classes.root}>
+                <div className={classes.scroll}>
+                    <div className={classes.history}>
+                        <h3>
+                            This conversation is not visible in the app because{' '}
+                            {Features[FeatureKeys.MultiUserChat].label} is disabled. Please enable the feature in the
+                            settings to view the conversation, or select a different or create a new conversation.
```

Comment:
```suggestion
                            settings to view the conversation, select a different one, or create a new conversation.
```

Reply:
this wasn't updated in your commit

Reply:
It's updated. You might have to select the latest diff.